podman_compose.py

```diff
@@ -855,8 +855,9 @@ async def assert_cnt_nets(compose, cnt):
     create missing networks
     """
     net = cnt.get("network_mode")
-    if net and not net.startswith("bridge"):
+    if net:
```

Comment:
Could you please tell me why is this change needed. It's not obvious to me. If there's some requirement in compose spec that I'm not aware of, please point me to it.

Reply:
When using bridge mode, podman-compose will use the default `podman` network.
We do not need to create this network, so just return as other network_mode.